plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -8,6 +13,25 @@ use alumet::{
 
 pub(crate) const CGROUP_MAX_TIME_COUNTER: u64 = u64::MAX;
 
+/// # Structure
+/// 
+/// * `CgroupV2Metric` - Public structure storing identifier of each CPU and memory data
+///
+/// # Parameters
+///
+/// * `name` - Name of a Kubernetes pod
+/// * `uid` - Identification of a Kubernetes pod
+/// * `namespace` - Path name of a Kubernetes pod
+/// * `node` - Kubernetes pod node
+/// * `time_used_tot` - Total CPU usage time by the cgroup
+/// * `time_used_user_mode` - CPU in user mode usage time by the cgroup
+/// * `time_used_system_mode` - CPU in system mode usage time by the cgroup
+/// * `anon_used_mem` - Anonyme used memory, corresponding to running process and various allocated memory
+/// * `file_mem` - Files memory, corresponding to open files and descriptors
+/// * `shared_mem` - Interprocess communication shared memory
+/// * `file_mapped_mem` - Mapped files in memory
+/// * `total_mem` - Total memory used by cgroup
+/// 
```

Comment:
Let's follow the Rust ecosystem's way of documenting. In particular:
- don't repeat the name of the struct at the beginning of the doc (it will be shown automatically)
- the first line should be a description, not a title (the title is automatically added by rustdoc)
- each field of the struct should be documented on its own, not in a list in the struct description